examples/shopify/README.md

```diff
@@ -24,6 +24,10 @@ This example includes a home page for listing products by category and a product
 
    Then head over to [Shopify](https://www.shopify.com/) and create a store. Here is [a good starting point](https://www.shopify.com/online) to get you started.
 
+   Note: This custom storefront example assumes all products are in stock, so make sure all products have "Track quantity" unchecked as pictured below.
```

Comment:
What happens if a product isn't in stock?

Reply:
you can't add it to the cart so "add to cart" looks like it succeeds, but it doesn't

Reply:
Got it. Is there a way to get information about if it's in stock from the API response and then disable the "add to cart" button in that case? Maybe you considered this and it wasn't worth the effort but I'm just curious

Reply:
Yeah this is possible. I just don't think it's worth it. 

Shopify requires you to enter a SKU and Barcode for items, so I just thought I would avoid it all together.
<img width="628" alt="CleanShot 2022-08-29 at 14 43 25@2x" src="https://user-images.githubusercontent.com/20950876/187284834-4eeb637a-2f97-472e-8c11-162d870b7ae1.png">